cobbler/settings/migrations/V3_2_1.py

```diff
@@ -135,7 +137,7 @@
         "yum_post_install_mirror": bool,
         "yumdownloader_flags": str,
     },
-    ignore_extra_keys=False,
+    ignore_extra_keys=True,
```

Comment:
Is there any reasons why we do this? I am not aware of any? This comment is valid for all versions.

Reply:
Hey Enno, I just provided small reasoning in the description:

Allow "extra keys" on settings: this prevent and issue when migrating settings from older version but you have some files at `settings.d` that are injecting settings from the newer Cobbler version you have just installed, as the files here provided by Cobbler are also processed as part of the old settings compilation to run the migration.

So, basically, installing new Cobbler 3.3.3 brings new files to `/etc/cobbler/settings.d`, and these new settings defined on those files are then evaluated when trying to migrate/validate old settings from a different version, as `settings.d` files will be evaluated as well when migrating the old settings.

Reply:
Yes. That still means that your settings need to be valid. `settings.d` was just a way of breaking up the main settings file. There were no additional keys defined in that folder and thus I still see no reason to change that.